docker/docker-compose.integration.yml

```diff
@@ -24,35 +24,27 @@ services:
       INFLUXDB_HTTP_LOG_ENABLED: 'true'
       INFLUXDB_CONTINUOUS_QUERIES_LOG_ENABLED: 'false'
 
-  influx_writer:
+  listenbrainz:
     build:
       context: ..
       dockerfile: Dockerfile
-    command: python3 -m "listenbrainz.influx_writer.influx_writer"
+      target: listenbrainz-dev
+    command: py.test listenbrainz/tests/integration
+    image: listenbrainz
```

Comment:
this has a build key _and_ an image key? what does this mean?

Reply:
This is to specify the name of the service image so it can be reused for the influx-writer service. (https://stackoverflow.com/a/35662191)

Reply:
cool, I hadn’t seen this behaviour before.